src/transformers/training_args.py

```diff
@@ -456,6 +459,12 @@ class TrainingArguments:
             )
         },
     )
+    save_on_each_node: bool = field(
+        default=False,
+        metadata={
+            "help": "When doing a multinode distributed training, whether to save once per node or just once on the main node."
```

Comment:
```suggestion
            "help": "When doing multi-node distributed training, whether to save models and checkpoints on each node, or only on the main one"
```

Reply:
hmm, should we note that it'll use the same filenames? and thus not to activate when one uses a shared fs?

Reply:
I'll add it in the docstring.